bundler/lib/dependabot/bundler/file_updater/requirement_replacer.rb

```diff
@@ -203,7 +203,7 @@ def serialized_req(req, use_equality_operator)
             # Gem::Requirement serializes exact matches as a string starting
             # with `=`. We may need to remove that equality operator if it
             # wasn't used originally.
-            tmp_req = tmp_req.gsub(/(?<![<>])=/, "") unless use_equality_operator
+            tmp_req = tmp_req.gsub(/(?<![<>!])=/, "") unless use_equality_operator
```

Comment:
Should we pull that regex out into a const? I notice we had to update it in two places 🤷